draft-ietf-cats-metric-definition.md

```diff
@@ -95,54 +102,64 @@ This document uses the following terms defined in {{!I-D.ietf-cats-framework}}:
 
 # Definition of Metrics
 
-Introducing a definition of metrics requires balancing the following trade-off: if the metrics are too fine-grained, they become unscalable due to the excessive number of metrics that must be communicated through the metrics distribution protocol. (See {{?I-D.rcr-opsawg-operational-compute-metrics}} for a discussion of metrics distribution protocols.) Conversely, if the metrics are too coarse-grained, they may lack the necessary information to make informed decisions. To ensure scalability while providing sufficient detail for effective decision-making, this document provides a definition of metrics that incorporates three levels of abstraction:
+## Problem Statement - Why Three Metric Levels?
+
+As is mentioned in {{I-D.ietf-cats-usecases-requirements}}, the resource model that describes CATS metrics MUST be scalable, which means the implementation is within accordable cost. Meanwhile, it MUST also be useful, that is a CATS system should choose appropriate metric(s) for instance selection, since metrics tend to impact differently on various use cases. 
+
```

Comment:
it is hard to say what is the meaning of useful, but I can understand. Good for now, we can review it later.